src/utils/ReduxUtils.ts

```diff
@@ -1,7 +1,21 @@
 import * as _ from 'underscore';
+import * as Redux from 'redux';
+import { connect } from 'react-redux';
 
 export class ReduxUtils {
   static mergeProps(stateProps: any, dispatchProps: any, ownProps: any) {
     return _.extend({}, stateProps, dispatchProps, ownProps);
   }
 }
+
+export function ReduxConnect(mapStateToProps?: any, mapDispatchToProps?: any, mergeProps?: any, options?: any): (target: any) => any {
+  return target => (connect(mapStateToProps, mapDispatchToProps, mergeProps, options)(target) as any);
+}
+
+export interface IReduxAction<T> extends Redux.Action {
+  payload?: T;
+}
+
+export interface IReduxProps {
```

Comment:
I stopped using dispatch in my components, but I will let the dispatch method in the interface since Redux is really passing it.